ui/lib/components/model/model.dart

```diff
@@ -72,41 +72,43 @@ extension ConversationUtil on g.Conversation {
   }
 
   static int mostRecentInboundFirst(g.Conversation c1, g.Conversation c2) {
-    var m1 = c1.mostRecentMessageInbound;
-    var m2 = c2.mostRecentMessageInbound;
-    if (m1 == null) {
-      if (m2 == null) {
-        return ConversationUtil.mostRecentMessageFirst(c1, c2);
-      } else {
-        return -1;
-      }
-    } else {
-      if (m2 == null) {
-        return 1;
-      } else {
-        // fall through
-      }
+    var d1 = c1.mostRecentMessageInbound?.datetime;
+    var d2 = c2.mostRecentMessageInbound?.datetime;
+
+    if (c1.docId == c2.docId) {
+      return 0;
+    }
+    if (d1 == d2) {
+      return c2.docId.compareTo(c1.docId);
```

Comment:
I think this should be
```suggestion
      return c1.docId.compareTo(c2.docId);
```
so that if the datetimes are the same, the comparison is alphabetical by ID?